tasks/main.yml

```diff
@@ -6,6 +6,7 @@
     - libgtk2.0-dev
     - libjasper-dev
     - libjpeg-dev
+    - liblapacke-dev
```

Comment:
Amazing @pkatseas! That's what we're looking for. 

Might be worth checking if this is maintained on a numpy reinstall, just to be sure that it won't link back to plain lapack